src/module/vue/components/attr-box.vue

```diff
@@ -18,26 +18,28 @@
 </template>
 
 <style lang="less" scoped>
-@import '../../../styles/mixins.less';
+@import (reference) '../../../styles/mixins.less';
+
 .stat {
-  &:before {
+  &::before {
```

Comment:
Oh boy. Was this broken, or is this just a preferred style?

Reply:
presently both work in most browsers (and a i'm pretty sure it's fine on any browser foundry officially supports), but "::before" appears to be the current standard while ":before" is an older style: https://developer.mozilla.org/en-US/docs/Web/CSS/::before